### ISSUE TITLE
Semantic Segmentation Loss Function

### ISSUE THREAD

**Issue by fatemeh-slh** [facebookarchive/caffe2#2514]
Hi,

I am trying to implement Deeplab model for semantic segmentation in Caffe2. I tried to use SigmoidCrossEntropyWithLogits as the loss function but I got this warning:

`WARNING:root:You are creating an op that the ModelHelper does not recognize: SigmoidCrossEntropyWithLogits.`

And it seems that the training process does not run correctly.

Does anybody try to implement a semantic segmentation (pre-trained on ImageNet) code in Caffe2?

Thanks.

**Comment by beichen2012** [comment]
the softmax and loss function is: spatial_softmax_with_loss_op.
